msteams-platform/toolkit/add-single-sign-on.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Add single sign-on to your Teams apps
+author: zyxiaoyuer
+description:  Describes Add single sign-on of Teams Toolkit
+ms.author: surbhigupta
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 05/16/2022
+---
+
+# Add Single sign-on
+
+Microsoft Teams provides single sign-on function for application to obtain signed-in Teams user token to access Microsoft Graph and other APIs. Teams Toolkit facilitates the interaction by removing some of the Azure AD flows and integrations behind some simple APIs. It enables you to add single sign-on (SSO) features easily to your Teams application.
+
+## Enable SSO support
+
+Teams Toolkit helps you to add SSO to the following Teams capabilities:
+
+* Tab
+* Bot
+* Notification bot: restify server
+* Command bot
+
+### Add SSO using Visual Studio Code
+
+The following steps helps you to add SSO using Teams Toolkit in Visual Studio Code
+
+1. Open **Microsoft Visual Studio Code**.
+2. Select Teams Toolkit :::image type="content" source="../assets/images/teams-toolkit-v2/add-sso/teams-toolkit-sidebar-icon.png" alt-text="sso add sidebar"::: from left navigation bar.
+3. Select **Add features** under **DEVELOPMENT**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/add-sso/sso-add features.png" alt-text="sso add features":::
+
+    * You can also open command palette and select **Teams: Add features**
+
+4. Select **Single Sign-On**.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/add-sso/sso-select features.png" alt-text="sso select":::
+
+### Add SSO using TeamsFx CLI
+
+You can run `teamsfx add sso`  command in your **project root directory**
+
+> [!Note]
+> The feature enables SSO for all existing applicable capabilities. Follow the same steps to enable SSO if you add capability later to the project.
+
+## Customize your project using Teams Toolkit
+
+The following table lists the changes Teams Toolkit makes to your project:
+
+   |**Type**|**File**|**Purpose**|
+   |--------|--------|-----------|
+   |Create|`aad.template.json` under `template\appPackage`|Azure AD application manifest represents your Azure AD app. `template\appPackage` helps to register an Azure AD app during local debug or provision stage.|
```

Comment:
can we change `\` to `/` to ensure consistency ?

Reply:
Updated as mentioned.

Reply:
Updated as mentioned.

Reply:
Updated as mentioned.